### DISCUSSION TITLE
长按是怎么写的

### DISCUSSION THREAD

**Discussion by i-learn-git** [gkd-kit/gkd#806] [Q&A] [answered]
长按是怎么写的

**Comment by lisonge** [comment, marked as answer]
https://gkd.li/api/interfaces/RawRuleProps#action

**Reply by i-learn-git** [reply]
{
  name: '公众号长按',
  key: 7,
  snapshotUrls: [
    'https://i.gkd.li/snapshot/18034319'
  ],
  rules: [
    {
      matches: [
        'View[text="订阅号消息"]'
      ]
    }
  ],action:"longClickCenter",
}
请问为什么这样写，保存后这个action后面的东西会自动消失？

**Reply by lisonge** [reply]
action 不支持写在 group 里要写在 rule 里

**Reply by lisonge** [reply]
```json5
{
name: '公众号长按',
key: 7,
snapshotUrls: [
'https://i.gkd.li/snapshot/18034319'
],
rules: [
{
matches: [
'View[text="订阅号消息"]'
],action:"longClickCenter",
}
]
}
```

**Reply by i-learn-git** [reply]
可以了，感谢
